packages/trip-form/src/__mocks__/modes-en.ts

```diff
@@ -194,4 +194,27 @@ const commonModes = {
   ]
 };
 
+// This variant of the data adds an option with a company as the first
+// option in the "CAR" category that has mixed modes ("CAR_PARK", "CAR_HAIL")
+export const modesWithCompanyFirstMixedCategory = {
+  ...commonModes,
+  categories: commonModes.categories.map(c => {
+    const nextCategory = { ...c };
```

Comment:
I think using a cloning method (e.g. `lodash.cloneDeep` used elsewhere in this repo) would be better and would better manipulate story-specific data. (see other comment about array below).

Reply:
Move the cloning/spreading inside the if block, so that if the car condition is not met, just return the original `c` object.

Reply:
Good call on the deep clone, will make that adjustment, the way I've set this up I'm re-using some parts of the `commonModes` tree and cloning some parts so probably best to disassociate them and clone the whole thing as you suggest.  Will look at adjusting the content of the `if` block as well

Reply:
Adjusted the creation of the new mock data to use `cloneDeep` and adjusted the `if` condition to work better with that approach